inc/options/class-wpseo-option-wpseo.php

```diff
@@ -119,6 +124,58 @@ public static function get_instance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Add filters to make sure that the option is merged with its defaults before being returned.
+	 *
+	 * @return void
+	 */
+	public function add_option_filters() {
+		parent::add_option_filters();
+
+		// Add filter to ensure the feature variables respect the network settings.
+		if ( has_filter( 'option_' . $this->option_name, array( $this, 'verify_features_against_network' ) ) === false ) {
+			add_filter( 'option_' . $this->option_name, array( $this, 'verify_features_against_network' ), 11 );
+		}
+	}
+
+	/**
+	 * Remove the option filters.
+	 * Called from the clean_up methods to make sure we retrieve the original old option.
+	 *
+	 * @return void
+	 */
+	public function remove_option_filters() {
+		parent::remove_option_filters();
+
+		remove_filter( 'option_' . $this->option_name, array( $this, 'verify_features_against_network' ), 11 );
+	}
+
+	/**
+	 * Add filters to make sure that the option default is returned if the option is not set.
+	 *
+	 * @return void
+	 */
+	public function add_default_filters() {
+		parent::add_default_filters();
+
+		// Don't change, needs to check for false as could return prio 0 which would evaluate to false.
+		if ( has_filter( 'default_option_' . $this->option_name, array( $this, 'verify_features_against_network' ) ) === false ) {
```

Comment:
You might consider to put `default_option_'  . $this->option_name` in a variable.